### PR TITLE
Updates code samples to include offset in print statements

### PR DESCRIPTION
**Pull request by dakota002**
@lpsinger do you have ideas on how to add the same bit to the c code sample?

### CODE REVIEW

**Comment by lpsinger** [comment]
> @lpsinger do you have ideas on how to add the same bit to the c code sample?

Sure. I'll make a PR to add it to your branch.

**Review by lpsinger — CHANGES_REQUESTED** [review]
Let's be explicit in the formatting of the message:

```
topic={topic} offset={offset}
```

**Comment by dakota002** [comment]
Good call, I can update that after your change

File: app/components/ClientSampleCode.tsx [hunk 1]
```diff
@@ -160,6 +162,7 @@ export function ClientSampleCode({
             await consumer.run({
               eachMessage: async (payload) => {
                 const value = payload.message.value
+                console.log(\`topic = \${payload.topic} offset = \${payload.message.offset}\`)
```
Comment (lpsinger) [inline app/components/ClientSampleCode.tsx:165]:
```suggestion
                console.log(\`topic=\${payload.topic}, offset=\${payload.message.offset}\`)
```

File: app/components/ClientSampleCode.tsx [hunk 2]
```diff
@@ -455,6 +459,7 @@ export function ClientSampleCode({
                 try
                 {
                   var consumeResult = consumer.Consume();
+                  Console.WriteLine( string.Format("topic = {0} offset = {1}",consumeResult.Topic, consumeResult.Offset));
```
Comment (lpsinger) [inline app/components/ClientSampleCode.tsx:462]:
```suggestion
                  Console.WriteLine(string.Format("topic={0}, offset={1}",consumeResult.Topic, consumeResult.Offset));
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Comment by lpsinger** [comment]
Have you run all of these code samples to test them, @dakota002?

**Comment by lpsinger** [comment]
@dakota002, here's the C language version: https://github.com/dakota002/gcn.nasa.gov/pull/202

**Comment by dakota002** [comment]
@lpsinger yep, I ran them to make sure I was using the correct methods for each language sample

**Review by lpsinger — APPROVED** [review]

**Pull request merged**